### DISCUSSION TITLE
Dúvida sobre Mult Tenant

### DISCUSSION THREAD

**Discussion by reluiz** [balta-io/forum#516] [Tira Dúvidas] [answered]
Bom dia! Estou estudando o curso de Aplicações Mult-Tenant com Entity Framework Core e fiquei com a seguinte dúvida: Quando um novo usuário for se cadastrar e criar a sua conta para usar o sistema precisamos também criar a empresa. Então, temos a classe "AccountController" onde temos a implementação do método que cria o cadastro do usuário e gostara de saber se seria correto nessa mesma classe "AccuntController" logo após criar o cadastro do usuário criar também o cadastro da empresa e em seguida atualizar o cadastro do usuário adicionando o CompanyId ?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @reluiz, como vai?

Seria sim, é um ótimo momento... inclusive pode colocar no mesmo formulário.

**Comment by reluiz** [comment]
Bom dia! Muito obrigdo @andrebaltieri.
